control/installation.xml

```diff
@@ -150,7 +152,7 @@
                       <proposed config:type="boolean">true</proposed>
                       <proposed_configurable config:type="boolean">true</proposed_configurable>
                       <!-- Disable it in second place if we don't fit in the disk -->
-                      <disable_order config:type="integer">2</disable_order>
+                      <disable_order config:type="integer">1</disable_order>
```

Comment:
Please update or delete the comments like "disable it in first place" or "disable it in second place". They assert the opposite to what the XML does, so it's confusing.